en/docs/develop/customizations/customizing-the-developer-portal/customize-developer-portal-credentials.md

```diff
@@ -0,0 +1,103 @@
+# Customize Developer Portal Credential Preferences
```

Comment:
Shall we add this doc under a section "security policy settings"
how about -> Customize the Developer Portal password policy settings ?

Reply:
Anway "Changing password" is not customization related operation. So I don't think the sections related to "password change" should go to a doc in  /customizations folder. 
Apart that we have this https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/identity-management-for-the-api-dev-portal/#password-policies where password policies are described.

And we don't have anything like "security policy settings" AFAIU; so we cannot use that phrase.